MBBSEmu/Program.cs

```diff
@@ -234,6 +236,23 @@ private void Run(string[] args)
                 var globalCache = _serviceResolver.GetService<IGlobalCache>();
                 var fileHandler = _serviceResolver.GetService<IFileUtility>();
 
+                //Setup Logger from AppSettings
+                LogManager.Configuration.LoggingRules.Clear();
+                LogManager.Configuration.AddRule(LogLevel.FromString(configuration.ConsoleLogLevel), LogLevel.Fatal, "consoleLogger");
+                if(!string.IsNullOrEmpty(configuration.FileLogName))
+                {
+                    var fileLogger = new FileTarget("fileLogger")
+                    {
+                        FileNameKind = 0,
+                        FileName = "${var:mbbsdir}" + configuration.FileLogName,
+                        Layout = Layout.FromString("${shortdate} ${time} ${level} ${callsite} ${message}"),
+                        DeleteOldFileOnStartup = true
```

Comment:
Do we want to make this optional/configurable/append by default? If someone crashed and restarted they'd lose the crash log.

Reply:
Removed this per our discussion